data/contact-info/1-safe-ride.yaml

```diff
@@ -1,11 +1,12 @@
-title: Safe Ride
+title: Safe Guard
 phoneNumber: '5077863666'
-buttonText: Request Safe Ride
+buttonText: Call Public Safety
```

Comment:
I think that if someone uses the app to call a safe ride, they won't be surprised when Pub Safe answers.